roles/neutron-common/templates/etc/neutron/l3_agent.ini

```diff
@@ -15,9 +13,7 @@ admin_user = neutron
 admin_password = {{ secrets.service_password }}
 metadata_ip = {{ endpoints.main }}
 use_namespaces = True
-{% if neutron.plugin == 'ml2' %}
 external_network_bridge =
-{% endif %}
```

Comment:
Since we're leaving conditionals in for ml2 to be the plugin elsewhere, we should probably leave this one in too. It'll make it easier if/when we add a different plugin.